src/app/components/button/button.css

```diff
@@ -0,0 +1,96 @@
+.btn {
+  cursor: pointer;
+  border: none;
+  opacity: 0.95;
+  padding: 0.8rem;
```

Comment:
People suggested me that when we have values less than 1rem then use px , I don't know if that is true or not I have not yet looked up that thing so I am just passing down the suggestion.

Reply:
padding I have used rem as I have to be responsive, yeah at other places like border box we need to use px.